source/backend/Paths.hx

```diff
@@ -253,7 +253,7 @@ class Paths
 			if(retVal != null) return retVal;
 		}
 
-		trace('oh no its returning null NOOOO ($file)');
+		trace('oh no its returning null NOOOO ($file) (' + posInfos.fileName + ', ' + posInfos.lineNumber + ')');
```

Comment:
instead try 
```suggestion
		FlxG.log.warn('Could not find image with key: "$key"' + (library == null ? "" : 'in library: "$library"'));
```

also you can have cleaner string interpolation with ${}
example:
```hx
trace('image not found ($file) (${posInfos.fileName}, ${posInfos.lineNumber})');
```
though i don't think, posInfos should be output in FlxG.log.warn

Reply:
i want to be clear, I am not a maintainer of this repo, just giving some feedback